src/main/java/org/apache/commons/cli/Options.java

```diff
@@ -231,8 +243,9 @@ public OptionGroup getOptionGroup(final Option opt) {
      * Gets the OptionGroups that are members of this Options instance.
      *
      * @return a Collection of OptionGroup instances.
+     * @since 1.7.0
      */
-    Collection<OptionGroup> getOptionGroups() {
+    public Collection<OptionGroup> getOptionGroups() {
```

Comment:
Hello @Claudenw 
The typing needs some thought I think if make this public: 
- First, do we really want this to be public?
- Why are we filtering out duplicates here? 
- If we indeed want to return a Set, why hide this and type the return as a Collection.
Consider:
- Changing the return type to a Set
- Changing the wrapping to an ArrayList

Thoughts?